ClassBuilder/immutable/Computer.java

```diff
@@ -0,0 +1,77 @@
+package immutable;
+
+public class Computer {
+
+    //required parameters
+    private String RAM;
+    private String HDD;

```

Comment:
should be lowerCamelCase
https://mate-academy.github.io/style-guides/java/java.html#s5.2.5-non-constant-field-names